test/WebJobs.Script.Tests/Managment/FunctionsSyncManagerTests.cs

```diff
@@ -1,110 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Script.WebHost.Management;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
-using Moq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests.Managment
-{
-    public class FunctionsSyncManagerTests
```

Comment:
Where did these test go?

Reply:
The original author is me. I noticed that this test should be in the integration testing.... so that I remove this and do the integration testing.